### ISSUE TITLE
cannot fork and clone repository

### ISSUE THREAD

**Issue by iamthechrisb** [christofmuc/JammerNetz#37]
I was trying to fork and clone repository to continue troubleshooting ASIO issues previously reported. this fails --  keep getting asked to authenticate to:
fatal: could not read Username for 'https://git.assembla.com': terminal prompts

**Comment by christofmuc** [comment]
True. Did you try to cancel the dialog? It should after a few cancels skip the checkout of portaudio, which seems to have moved behind a user login. We're not using it directly, but via the Q library. I will try to update Q and see if that fixes the problem!

**Comment by christofmuc** [comment]
I pushed the new Q and changes to the build to https://github.com/christofmuc/JammerNetz/tree/ASIOdebugging the branch. Can you check that this works for you? It should no longer try to clone portaudio, because Q removed it.

**Comment by christofmuc** [comment]
So you'd clone non-recursively, switch the branch, then fetch the submodules:

1. git clone https://github.com/christofmuc/JammerNetz
2. cd JammerNetz
3. git switch ASIOdebugging
4. git submodule --init- --recursive

should do it.

**Comment by iamthechrisb** [comment]
I'll do some checking soon. thanks for the reply. While you are around, is it possible for you to update to Juice 8x?

**Comment by christofmuc** [comment]
Sure thing, did it in the branch as well

**Comment by iamthechrisb** [comment]
I can fork now. thank you. is there a chance you will update the client build with updated juice?

**Comment by christofmuc** [comment]
I just released a 2.3.0 release candidate, tested only on Windows so far. This contains the ASIO debugging code and the manual start/stop thing, in addition to a new MIDI clock feature.
